### PR TITLE
[WCM] Using api.web3modal.com

### PR DESCRIPTION
**Pull request by radeknovis**
Replaces explorer api with api.web3modal.com. Also includes change to paginate response

### CODE REVIEW

**Comment by christianlantin** [comment]
Master

File: Example/DApp/Modules/Sign/SignPresenter.swift [hunk 1]
```diff
@@ -52,6 +53,16 @@ final class SignPresenter: ObservableObject {
         Web3Modal.present(from: nil)
     }
     
+    func connectWalletWithWCM() {
+        Task {
+            WalletConnectModal.set(sessionParams: .init(
```
Comment (flypaper0) [inline Example/DApp/Modules/Sign/SignPresenter.swift:58]: I do not see async context here, do you need Task?

File: Sources/WalletConnectModal/Modal/Screens/WalletDetail/WalletDetailViewModel.swift [hunk 2]
```diff
@@ -15,49 +15,44 @@ final class WalletDetailViewModel: ObservableObject {
...
     ) {
         self.wallet = wallet
         self.deeplinkHandler = deeplinkHandler
-        preferredPlatform = wallet.app.ios != nil ? .native : .browser
+        preferredPlatform = wallet.appStore != nil ? .native : .browser
     }
     
     func handle(_ event: Event) {
         switch event {
         case .onAppear:
```
Comment (flypaper0) [inline Sources/WalletConnectModal/Modal/Screens/WalletDetail/WalletDetailViewModel.swift:38]: `case .onAppear, .didTapUniversalLink, . didTapTryAgain:`

File: Sources/WalletConnectModal/Modal/Screens/WalletList.swift [hunk 3]
```diff
@@ -1,16 +1,45 @@
...
+        self.onWalletTap = onWalletTap
+        self.numberOfColumns = numberOfColumns
+        self.availableSize = availableSize
+        self.infiniteScrollLoading = infiniteScrollLoading
+        
+        if #available(iOS 14.0, *) {
+            // iOS 14 doesn't have extra separators below the list by default.
+        } else {
+            // To remove only extra separators below the list:
+            UITableView.appearance().tableFooterView = UIView()
```
Comment (flypaper0) [inline Sources/WalletConnectModal/Modal/Screens/WalletList.swift:36]: Can it affect client's app?

File: Sources/WalletConnectModal/Modal/Screens/WalletList.swift [hunk 4]
```diff
@@ -1,16 +1,45 @@
...
+        }
+
+        // To remove all separators including the actual ones:
+        UITableView.appearance().separatorStyle = .none
```
Comment (flypaper0) [inline Sources/WalletConnectModal/Modal/Screens/WalletList.swift:40]: same question

**Pull request merged**